src/main/java/core/basesyntax/service/impl/ContentWriterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ContentWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ContentWriterImpl implements ContentWriter {
```

Comment:
same as in reader
don't forget to add exception object to `throw` clause

Reply:
ok